packages/swap/src/routes/v2/utils.ts

```diff
@@ -405,6 +428,21 @@ export const getDcaParams = (
       }
     : undefined;
 
+export const getCcmParams = (
+  swapRequest: Awaited<ReturnType<typeof getLatestSwapForId>>['swapRequest'],
+  swapDepositChannel: Awaited<ReturnType<typeof getLatestSwapForId>>['swapDepositChannel'],
+) => {
+  const ccmGasBudget = readField(swapRequest, swapDepositChannel, 'ccmGasBudget');
+  const ccmMessage = readField(swapRequest, swapDepositChannel, 'ccmMessage');
+
+  return ccmGasBudget || ccmMessage
```

Comment:
why not additional data?

Reply:
im not sure if we are event indexing that atm 😬